src/ethereum_test_specs/blockchain.py

```diff
@@ -482,6 +483,7 @@ def generate_block_data(
             # Modify any parameter specified in the `rlp_modifier` after
             # transition tool processing.
             header = block.rlp_modifier.apply(header)
+            header.fork = fork  # Deleted during `apply` because `exclude=True`
```

Comment:
```suggestion
            header.fork = fork  # Deleted during `apply` because `exclude_none=True`
```

Reply:
https://github.com/ethereum/execution-spec-tests/blob/6e4c3a1119039bd79058b56c4b28564fb2ee090c/src/ethereum_test_fixtures/blockchain.py#L122

it's actually full `exclude=True` in this instance.

Reply:
Got it, thanks!